src/atoms/LiistBttn.js

```diff
@@ -45,18 +38,18 @@ export class LiistBttn extends LitElement {
 
   constructor() {
     super();
+    this.text = "loading...";
     this.color = "grey";
     this.bgColor = "#ccc";
     this.width = "25";
-
   }
 
   render() {
     return html`
-      <div class="container">
-        <a href="${this.url}">
+      <div>
+        <a href="${this.url || "#" }">
           <div class="button" style="width:${this.width}%; background-color: ${this.bgColor};">
-            <p style="color:${this.color}"><slot name="text">loading ...</slot></p>
+            <p style="color:${this.color}">${this.text}</p>
```

Comment:
I removed the slot in the button, because it seems easier to write `<liist-bttn text="my button text">` instead of opening a html tag and giving it the attribute `name="slotname"`. Also the advantage is that without slot we can change the text property (with js) and the component rerenders automatically. 
```